move/version_control/sources/version_control.move

```diff
@@ -65,8 +65,8 @@ public fun push_back(self: &mut VersionControl, function_names: vector<vector<u8
 ///     // do the thing.
 /// }
 /// ```
-public fun check(self: &VersionControl, version: u64, function: vector<u8>) {
```

Comment:
`new` and `push_back` should be updated to accept strings as well. You can use `map` as I mentioned to convert bytes to string before calling the new method